### PR TITLE
doc: add meeting minutes 2017-05-10

### PR DESCRIPTION
**Pull request by joshgav**
cc @nodejs/ctc @matteocollina @joyeecheung

### CODE REVIEW

**Review by jasnell — APPROVED** [review]
Rubber stamp LGTM

**Review by joyeecheung — APPROVED** [review]

**Review by thefourtheye — APPROVED** [review]

**Review by mhdawson — APPROVED** [review]
LGTM

**Review by benjamingr — APPROVED** [review]

File: meetings/2017-05-10.md [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+
+## Upcoming Meetings
+
+* **Node.js Foundation Calendar**: <https://calendar.google.com/calendar/embed?src=nodejs.org_nr77ama8p7d7f9ajrpnu506c98%40group.calendar.google.com>
+
+Click `+GoogleCalendar` at the bottom right to add to your own Google calendar.
+
+* **TC39 Meetup**: May 22,
+  [CTC#67](https://github.com/nodejs/CTC/issues/67)        
+* **CTC**: May 17, 2000 UTC
```
Comment (TimothyGu) [inline meetings/2017-05-10.md]: I've always been very confused by this line… is "2000" supposed to be the time of day, i.e. "20:00" or "8:00 PM"?

Comment (Trott) [reply meetings/2017-05-10.md]: Yes, that's right. Would it be more clear if we started writing it `20:00` instead? Also I'm never sure if it should be `UTC 20:00` or `20:00 UTC`. Not sure if it even matters.

Comment (gibfahn) [reply meetings/2017-05-10.md]: `May 17, 20:00 UTC` would be ideal IMO, currently `2000` looks like the year.

A quick google suggests that UTC should come after.

**Pull request merged**